games/r.yaml

```diff
@@ -153,20 +154,21 @@
 
 - name: Robotfindskitten
   meta:
-    genre: [simulation]
+    genre: [Adventure, Roguelike]
```

Comment:
I think this is better suited as a simulation-roguelike game. Or just roguelike, doesn't look like an adventure game to me.